cypress/integration/examples/pear_spec.js

```diff
@@ -0,0 +1,201 @@
+  })
+
+  it('As a user, when I navigate to different areas of the application, the URL should update accordingly.', () => {
+    cy.visit('http://localhost:3000/new')
+    cy.get('H2')
+    .contains('Submit a new request')
+    cy.visit('http://localhost:3000/dashboard')
+    cy.get('H1')
+    .contains('Active Reviews')
+    cy.visit('http://localhost:3000/')
+    cy.get('H2')
+    .contains('Open Review Requests')
+  })
+
+  it('As a user, when I select a filter value, I will only be shown requests relevant to that filter', () => {
+    cy.get('#languageFilter').select('Java')
+    cy.get('.card-container')
+    .children()
+    .should('have.length', 1)
+  })
+})
+
+
+describe('Reviews', () => {
+  before(() => {
+      cy.fixture('mockReviews.json')
+      .then(reviewData => {
+          cy.intercept('GET', 'http://localhost:3003/api/v1/reviews', reviewData)
+      })
+
+
+      cy.intercept({
+        method: 'PUT',
+        url: 'http://localhost:3003/api/v1/reviews/accept/22/Jackson'
+      },
+        {
+          statusCode: 200,
+          body:
+          [{"username":"JeffShepherd","summary":"this is a summary","email":"mynamejeff@yahoo.com","language":"Python","date":"02/24/21","repo":"https://github.com/JeffShepherd/Rancid-Tomatillos","status":"active","reviewer":"Jackson","id":1},
+          {"username":"jacksonmcguire","summary":"this is a summary","email":"lumberjack@gmail.com","language":"JavaScript","date":"02/24/21","repo":"https://github.com/Jacksonmcguire/intention-timer","status":"active","reviewer":"Jackson","id":22},
+          {"username":"josharagon","summary":"this is a summary","email":"joshy@gmail.com","language":"C++","date":"02/24/21","repo":"https://github.com/josharagon/self-care-center","status":"complete","reviewer":"Jackson","id":333},
+          {"username":"owenhallgren","summary":"this is a summary","email":"owenhallgren@hotmail.com","language":"C#","date":"02/24/21","repo":"https://github.com/owenhallgren/js-fun-at-the-library","status":"","reviewer":"","id":4444},
+          {"username":"rdtho2525","summary":"this is a summary","email":"reggieveggie@msn.com","language":"Ruby","date":"02/01/21","repo":"https://github.com/JeffShepherd/Rancid-Tomatillos","status":"complete","reviewer":"Jackson","id":5050},
+          {"username":"aemiers","summary":"this is a summary","email":"ispeakgerman@gmail.com","language":"Java","date":"06/15/21","repo":"https://github.com/Jacksonmcguire/intention-timer","status":"","reviewer":"","id":6060},
+          {"username":"ConnorAndersonLarson","summary":"this is a summary","email":"cal@gmail.com","language":"PHP","date":"09/18/21","repo":"https://github.com/josharagon/self-care-center","status":"","reviewer":"","id":123},
+          {"username":"Shakikka","summary":"this is a summary","email":"shakika@hotmail.com","language":"Other","date":"02/24/21","repo":"https://github.com/owenhallgren/js-fun-at-the-library","status":"active","reviewer":"Jackson","id":777}
```

Comment:
Consider creating a fixture/variable representing this data as it is repeated a few times.